src/components/Overlay.js

```diff
@@ -161,14 +180,8 @@ export default class Overlay extends React.Component {
     };
 
     return (
-      <div
```

Comment:
Do not change code you don't need.

Reply:
You mean the JSX indentation? This came from prettier I fear. Let's revert this.